src/viam/sdk/services/mlmodel/server.cpp

```diff
@@ -91,8 +91,8 @@ ::grpc::Status MLModelServiceServer::Metadata(
             target.Reserve(source.size());
             for (auto&& s : source) {
                 auto& new_entry = *target.Add();
-                *new_entry.mutable_name() = std::move(s.name);
-                *new_entry.mutable_description() = std::move(s.description);
+                *new_entry.mutable_name() = s.name;
+                *new_entry.mutable_description() = s.description;
```

Comment:
Arguably, the correct fix here and below is to make the `source` parameter non-const.